io-engine/src/bdev/nvmx/uri.rs

```diff
@@ -329,10 +334,10 @@ impl CreateDestroy for NvmfDeviceTemplate {
             name: self.get_name(),
         };
 
-        let poller = poller::Builder::new()
+        let poller = PollerBuilder::new()
             .with_name("nvme_async_probe_poller")
-            .with_interval(1000) // poll every 1 second
-            .with_poll_fn(move || unsafe {
+            .with_interval(Duration::from_micros(1000)) // poll every 1 ms
```

Comment:
```suggestion
            .with_interval(Duration::from_millis(1))
```
Now that we have proper type we don't need the comment?

Reply:
The previous comment was incorrect: with previous poller incarnation, .with_interval() was always in mircosec. The comment said "every 1 sec", but in fact it polled every 1 millisec. So I preserved the comment to show that this change wasn't a mistake.

Reply:
I will fix comment.